web/src/components/button/ButtonType/CustomFields/CardButtonCustomFields/index.tsx

```diff
@@ -13,7 +14,7 @@ export function CardButtonCustomFields({ customFields, button }) {
   );
 
   const renderFields = () => {
-    const reversedCustomFields = [...customFields]
+    const reversedCustomFields = [...customFields].reverse();
```

Comment:
what is this reverse doing? should we be able to order by name?

Reply:
I want date and place to be at the bottom, after other fields